src/lib.rs

```diff
@@ -51,7 +51,7 @@ use std::process::ExitStatus;
 pub fn that<T: AsRef<OsStr> + Sized>(path: T) -> io::Result<ExitStatus> {
     let path_ref = path.as_ref();
     let mut last_err: io::Error = io::Error::from_raw_os_error(0);
-    for program in &["xdg-open", "gnome-open", "kde-open"] {
+    for program in &["wslopen", "xdg-open", "gnome-open", "kde-open"] {
```

Comment:
Something that came to my mind: Is `wslopen` the most likely candidate on linux systems? Ideally, these are sorted by likelyhood of being present on the system to have the least amount of misses.
Without actual statistics I would vote for putting the `wslopen` program to the end of the array.
What do you think?